dice/dice.go

```diff
@@ -39,7 +39,7 @@ func NewDice(sides int) *Dice {
 func NewDiceSlice(count int, sides int) []*Dice {
 	// Seeds once, rather than per-dice
 	rand.Seed(time.Now().UnixNano())
-	dice := make([]*Dice, count)
+	dice := make([]*Dice, 0, count)
```

Comment:
Would we get the same thing if we did:
```go
var dice []*Dice
```

Reply:
Not actually sure, but with the capacity, it will make sure the buffer is created to fit exactly the count wanted. Can't remember if you can append to a slice, that is not initialised, and if it were to work, if it will allocate more than required.